Engine/ResourceManagement/Importer/ModelImporter.cpp

```diff
@@ -41,6 +42,7 @@ ModelImporter::~ModelImporter()
 
 FileData ModelImporter::ExtractData(Path& assets_file_path, const Metafile& metafile) const
 {
+	const ModelMetafile& model_metafile = static_cast<const ModelMetafile&>(metafile);
```

Comment:
maybe change the name for something like temporary_model_metafile or so because you have the same name in the header, or it is intentionally since it is a temporal var?

Reply:
The name in the header is inside current_model_data so no problem with it.